packages/repo-collector-types/src/index.ts

```diff
@@ -125,6 +137,10 @@ export interface WebappMetricDataRepoDatapoint {
       browseUrl: string
     }[]
   }
+  sonarCloud?: {
```

Comment:
nit: `sonarCloud?` -> `sonarCloud`? As far as I can tell it is always explicitly set to a non-null value.

Reply:
True- it will be undefined, not null

Reply:
Just to clarify a bit - when you mark a property with `?` it means that the property can be `undefined`. In this case that's not really the case, is it? https://github.com/capralifecycle/liflig-repo-metrics/blob/57b7f3c943551aec03f226c4d4e24b5ea4dfdd07/packages/repo-collector/src/webapp/webapp.ts#L119 always sets this property to a value, so we should be able to mark the property as required. This is not a big deal, but it makes the type more accurate.